web/src/hooks/useSubmitPersonalInfo.js

```diff
@@ -19,9 +21,13 @@ export const useSubmitPersonalInfo = () => {
       });
 
       const data = await response.json();
-
       if (response.ok) {
         setSuccessMessage(data.message);
+        setCvData(data.CV);
+        saveFormData(data.CV);
```

Comment:
do we need both of these? ideally we should only use the `saveFormData` as they both perform the same action